workshops/Convert text to speech/README.md

```diff
@@ -0,0 +1,128 @@
+The gTTS API supports several languages including English, Hindi, Tamil, French, German, and many more. The speech can be delivered in any one of the two available audio speeds, fast or slow. However, as of the latest update, it is not possible to change the voice of the generated audio.
+
+For getting this module you need to install the gtts package. Don't worry if you are using repl.it.In repl.it it will be automatically installed when you run the code.
+But if you are using another platform you need to install the packages via terminal
+
+To install the gTTS API, open terminal/cmd and write
+
+`pip install gTTS`
+
+Now we are all set to write the code.
+
+## Lets code
+
+We need to import the module we installed.
+
+```python
+from gtts import gTTS
+```
+
+We are just importing the modules in this line.
+
+```python
+mytext = str(input("Enter the text you want to convert :-"))
+```
+
+In this line, we need to pass the string which we need to convert to speech. Here mytext is a variable that holds our text. When the code runs it will ask for the user's input to enter a text.
+
+<img width="1440" alt="Screenshot 2020-09-18 at 5 39 06 PM" src="https://cloud-mvq9zgpof.vercel.app/0screenshot_2020-10-10_at_11.06.28_am.png">
+
+```python
+language = str(input("Enter the language code for example 'en' for English :-"))
+```
+
+In this line, we are assigning the language to the variable language. Gtts API supports multiple languages but here we are going with English. Every language has its short forms for English it is en. For getting other language code google. So we are declaring the language variable as 'en'. So the text will be converted in the language specified. When the code runs it will ask for the user's input to enter a language.
+
+<img width="1440" alt="Screenshot 2020-09-18 at 5 39 06 PM" src="https://cloud-8uqrl4ft9.vercel.app/0screenshot_2020-10-10_at_11.07.12_am.png">
+
+```python
+myobj = gTTS(text=mytext, lang=language, slow=False)
+```
+
+Here in this line we are declaring another variable 'myobg' and we are giving it the value as calling the module we imported and the gtts is a function here. It takes 3 arguments one is the text. Here we have declared our text in a variable before we need to give it here. Then the language. We also assigned a variable and collected which language we want we also need to specify the variable name here. Then comes the speed you want slow = False means the voice will be faster. Slow=True means the voice will be slow.
+
+```python
+myobj.save("voice.mp3")
+```
+
+So before the above code, the text is converted to the voice but we need to save it right? To save it, the gtts module itself has a save function. the variable we converted to voice .save that's it And also give a name for the file which will be saved here it is voice.mp3.Hope you got it.
```

Comment:
“Hope you got it”—it’s really important that we use a welcoming tone for beginners